### ISSUE TITLE
AttributeError: module 'numpy' has no attribute 'float'.

### ISSUE THREAD

**Issue by woctezuma** [woctezuma/SimSwap-colab#20]
```bash
!python test_wholeimage_swapsingle.py
```
```
/usr/local/lib/python3.10/dist-packages/onnxruntime/capi/onnxruntime_inference_collection.py:69: UserWarning: Specified provider 'CUDAExecutionProvider' is not in available provider names.Available providers: 'AzureExecutionProvider, CPUExecutionProvider'
  warnings.warn(
Downloading: "https://download.pytorch.org/models/resnet18-5c106cde.pth" to /root/.cache/torch/hub/checkpoints/resnet18-5c106cde.pth
100% 44.7M/44.7M [00:00<00:00, 227MB/s]
/usr/local/lib/python3.10/dist-packages/torch/functional.py:504: UserWarning: torch.meshgrid: in an upcoming release, it will be required to pass the indexing argument. (Triggered internally at ../aten/src/ATen/native/TensorShape.cpp:3526.)
  return _VF.meshgrid(tensors, **kwargs)  # type: ignore[attr-defined]
Traceback (most recent call last):
  File "/content/SimSwap/test_wholeimage_swapsingle.py", line 105, in <module>
    reverse2wholeimage(b_align_crop_tenor_list, swap_result_list, b_mat_list, crop_size, img_b_whole, logoclass, \
  File "/content/SimSwap/util/reverse2original.py", line 157, in reverse2wholeimage
    target_image = np.array(target_image, dtype=np.float) * 255
  File "/usr/local/lib/python3.10/dist-packages/numpy/__init__.py", line 319, in __getattr__
    raise AttributeError(__former_attrs__[attr])
AttributeError: module 'numpy' has no attribute 'float'.
`np.float` was a deprecated alias for the builtin `float`. To avoid this error in existing code, use `float` by itself. Doing this will not modify any behavior and is safe. If you specifically wanted the numpy scalar type, use `np.float64` here.
The aliases was originally deprecated in NumPy 1.20; for more details and guidance see the original release note at:
    https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations. Did you mean: 'cfloat'?
```

```bash
!python test_wholeimage_swapmulti.py
```
```
/usr/local/lib/python3.10/dist-packages/onnxruntime/capi/onnxruntime_inference_collection.py:69: UserWarning: Specified provider 'CUDAExecutionProvider' is not in available provider names.Available providers: 'AzureExecutionProvider, CPUExecutionProvider'
  warnings.warn(
/usr/local/lib/python3.10/dist-packages/torch/functional.py:504: UserWarning: torch.meshgrid: in an upcoming release, it will be required to pass the indexing argument. (Triggered internally at ../aten/src/ATen/native/TensorShape.cpp:3526.)
  return _VF.meshgrid(tensors, **kwargs)  # type: ignore[attr-defined]
Traceback (most recent call last):
  File "/content/SimSwap/test_wholeimage_swapmulti.py", line 106, in <module>
    reverse2wholeimage(b_align_crop_tenor_list,swap_result_list, b_mat_list, crop_size, img_b_whole, logoclass, \
  File "/content/SimSwap/util/reverse2original.py", line 157, in reverse2wholeimage
    target_image = np.array(target_image, dtype=np.float) * 255
  File "/usr/local/lib/python3.10/dist-packages/numpy/__init__.py", line 319, in __getattr__
    raise AttributeError(__former_attrs__[attr])
AttributeError: module 'numpy' has no attribute 'float'.
`np.float` was a deprecated alias for the builtin `float`. To avoid this error in existing code, use `float` by itself. Doing this will not modify any behavior and is safe. If you specifically wanted the numpy scalar type, use `np.float64` here.
The aliases was originally deprecated in NumPy 1.20; for more details and guidance see the original release note at:
    https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations. Did you mean: 'cfloat'?
```

**Comment by woctezuma** [comment]
The error arises at:
- https://github.com/woctezuma/SimSwap/blob/037c7f7a60b747c0b25d7c58e977edd2a8d15276/util/reverse2original.py#L157

These other lines would raise the same error:
- https://github.com/woctezuma/SimSwap/blob/037c7f7a60b747c0b25d7c58e977edd2a8d15276/util/reverse2original.py#L159
- https://github.com/woctezuma/SimSwap/blob/037c7f7a60b747c0b25d7c58e977edd2a8d15276/util/reverse2original.py#L168

**Comment by woctezuma** [comment]
Fixed with [`febc35a`](https://github.com/woctezuma/SimSwap/commit/febc35aee877b2badc15ea2690c20f78ef16a890).
